templates/test/cpp/naming/resolver_component_tests_defs.include

```diff
@@ -18,12 +18,12 @@
 set -ex
 
 # all command args required in this set order
-FLAGS_test_bin_path=$(echo "$1" | grep '\--test_bin_path=' | cut -d "=" -f 2)
-FLAGS_dns_server_bin_path=$(echo "$2" | grep '\--dns_server_bin_path=' | cut -d "=" -f 2)
-FLAGS_records_config_path=$(echo "$3" | grep '\--records_config_path=' | cut -d "=" -f 2)
-FLAGS_dns_server_port=$(echo "$4" | grep '\--dns_server_port=' | cut -d "=" -f 2)
-FLAGS_dns_resolver_bin_path=$(echo "$5" | grep '\--dns_resolver_bin_path=' | cut -d "=" -f 2)
-FLAGS_tcp_connect_bin_path=$(echo "$6" | grep '\--tcp_connect_bin_path=' | cut -d "=" -f 2)
+FLAGS_test_bin_path=$(echo "$1" | grep '\--test_bin_path=' | sed 's/^--test_bin_path=//')
```

Comment:
a more idiomatic way of doing this would be `sed /^--test_bin_path=/d'. Same for the other lines.

Reply:
I am giving this a try, but `sed /^--test_bin_path=/d'` will delete the full string, no?

Reply:
Ok yes, that was a dumb suggestion, sorry. Approved.